src/open_inwoner/pdc/admin/product.py

```diff
@@ -62,6 +62,10 @@ def __init__(self, *args, **kwargs):
                     "categories"
                 ].initial = self.instance.categories.intersection(user_categories)
 
+        self.fields["form"].choices = Product.form.field.get_choices(
+            include_blank=Product.form.field.blank
+        )
```

Comment:
Is there no other way to do this? It would be nice if the library itself dealt with this

Reply:
Yeah, this is a bit odd. The library currently handles the field choices on the model level, hence the problem and this hackish workaround. I'll put the PR in draft for the time being and look into improving this in the library.